images/varnish/bootstrap.sh

```diff
@@ -21,6 +21,9 @@ service varnishncsa start
 
 varnish-agent
 
+# Hook to dispatch logs in graylog (Provisional, awaiting smarter & cleaner config)
+/usr/bin/varnishncsa -a -P /var/run/varnishncsa/varnishncsa.pid -F "{\"method\": \"%m\", \"X-Match\": \"%{X-Match}i\", \"X-Location-Id\": \"%{X-Location-Id}i\"}" | grep -P --line-buffered "(BAN|PURGE)" | nc -w 1 -u graylog-server 5555 &
```

Comment:
Varnishncsa service is already launched. Let's try to use the existing daemon and parse needed fields in existing log file.